.github/workflows/windows-package.yml

```diff
@@ -41,9 +41,10 @@ jobs:
           ls -l
           python setup.py develop
           py.test -v
-    - run: |
-          python setup.py bdist_wheel
+          
       if: ${{ github.event_name == 'release' }}
```

Comment:
`if` should be below `run`

Reply:
otherwise it might be considered as part of the previous command

Reply:
ok